scripts/build.ts

```diff
@@ -39,3 +45,25 @@ function buildPackage() {
     encoding: "utf-8",
   });
 }
+
+function buildExtendedJSON() {
+  // TODO: Validate the resulting JSON against a schema.
+  for (const [id, featureData] of Object.entries(data.features)) {
+    if (Array.isArray(featureData.compat_features) && featureData.status) {
+      const by_compat_key = {};
+
+      for (const key of featureData.compat_features) {
+        by_compat_key[key] = { status: getStatus(id, key) };
```

Comment:
I think this has to be unwrapped to match the schema:

```suggestion
        by_compat_key[key] = getStatus(id, key);
```